proposals/intptr-operators.md

```diff
@@ -0,0 +1,54 @@
+# Operators should be exposed for `System.IntPtr` and `System.UIntPtr`
+
+* [x] Proposed
+* [ ] Prototype: [Complete](https://github.com/PROTOTYPE_OWNER/roslyn/BRANCH_NAME)
+* [ ] Implementation: [In Progress](https://github.com/dotnet/roslyn/BRANCH_NAME)
+* [ ] Specification: [Not Started](pr/1)
```

Comment:
I think having the standard links defined here as part of a proposal is a good thing, perhaps use a strike through and a \\ to break the link:

    ~~[Complete]\(https://github.com/PROTOTYPE_OWNER/roslyn/BRANCH_NAME)~~

rendering as ~~[Complete]\(https://github.com/PROTOTYPE_OWNER/roslyn/BRANCH_NAME)~~